plugins/woocommerce/tests/legacy/unit-tests/session/class-wc-tests-session-handler.php

```diff
@@ -231,41 +253,184 @@ public function test_destroy_session_user_mismatch() {
 	}
 
 	/**
-	 * @testdox Test that session is destroyed if the session data is empty and the cart is not yet initialized.
+	 * Test that method destroys session when all conditions are met.
```

Comment:
Why have you removed the [`@testdox`](https://docs.phpunit.de/en/10.5/annotations.html#testdox) annotation here? In fact it would be good to add it to any new test written, since that improves readability of error messages when tests fail (try running phpunit with the `--testdox` argument).